frontend/src/components/landingPage/MoreInformation.vue

```diff
@@ -0,0 +1,136 @@
+<template>
```

Comment:
Currently it is hard to see that the page is not done loading btw... Maybe open an issue for this.

Reply:
#65 Issue has been raised for the same.